central/processlisteningonport/datastore/datastore_impl_test.go

```diff
@@ -140,27 +142,29 @@ var (
 		Protocol:       storage.L4Protocol_L4_PROTOCOL_TCP,
 		CloseTimestamp: nil,
 		Process: &storage.ProcessIndicatorUniqueKey{
-			PodId:               fixtureconsts.PodUID1,
+			PodId:               fixtureconsts.PodName1,
```

Comment:
why the switch from UID1 to Name1?

Reply:
The PodId is actually the pod name. There is inconsistent naming in the database and the code. I think most places with PodId should be changed to PodName. In the pod table the name of the pod is in the name column. In most other places the name of the pod is PodId, which is confusing. I created the PodName fixtures in the hopes that it will be used in the code for pods. For example here https://github.com/stackrox/stackrox/blob/master/pkg/fixtures/pod.go#L11

In the location Id should be a UUID and there should be an entry for Name, which should be set to fixtureconsts.PodName1. I suspect that there are no proper tests that use the fixture that I linked to. It is probably just used by mocks. In another PR where I tried to add that fixture to a database it didn't work, because Id wasn't a UUID.

I think PodId should be switched to PodName in other PRs. I think doing it here would cause difficulties.

Here is an example response from the listening endpoints API

```
        {
            "endpoint": {
                "port": 9090,
                "protocol": "L4_PROTOCOL_TCP"
            },
            "deploymentId": "a7568b76-235d-4d42-939f-69ef9ea76a02",
            "containerName": "central",
            "podId": "central-5d88f6fcb5-vt6qt",
            "podUid": "",
            "signal": {
                "id": "",
                "containerId": "",
                "time": null,
                "name": "central",
                "args": "",
                "execFilePath": "/stackrox/central",
                "pid": 0,
                "uid": 0,
                "gid": 0,
                "lineage": [],
                "scraped": false,
                "lineageInfo": []
            },
            "clusterId": "",
            "namespace": "",
            "containerStartTime": null,
            "imageId": ""
        }
```
You can see that the PodId is the name of the pod, not its UID.